examples/magento-graphcms/pages/blog/[url].tsx

```diff
@@ -80,14 +82,14 @@ export const getStaticPaths: GetPageStaticPaths = async ({ locales = [] }) => {
 
 export const getStaticProps: GetPageStaticProps = async ({ locale, params }) => {
   const urlKey = params?.url ?? '??'
+
   const client = graphqlSharedClient(locale)
   const staticClient = graphqlSsrClient(locale)
   const limit = 4
   const conf = client.query({ query: StoreConfigDocument })
-  const page = staticClient.query({
-    query: DefaultPageDocument,
-    variables: { url: `blog/${urlKey}` },
-  })
+
+  const withoutDynamic = hygraphPageContent(staticClient, `blog/${urlKey}`)
+  const page = hygraphDynamicContent(staticClient, withoutDynamic, {})
```

Comment:
hygraphDynamicContent with the third argument empty doen't make sense as this will never add any dynamic blocks, right?